base/threadingconstructs.jl

```diff
@@ -110,15 +111,73 @@ A barrier is placed at the end of the loop which waits for all tasks to finish
 execution.
 
 The `schedule` argument can be used to request a particular scheduling policy.
-The only currently supported value is `:static`, which creates one task per thread
-and divides the iterations equally among them. Specifying `:static` is an error
-if used from inside another `@threads` loop or from a thread other than 1.
+
+Except for `:static` scheduling, how the iterations are assigned to tasks, and how the tasks
+are assigned to the worker threads are undefined. The exact assignments can be different
+for each execution. The scheduling option should be considered as a hint. The loop body
```

Comment:
```suggestion
for each execution. The scheduling option shall be considered a hint. The loop body
```

?considered as a hint? by whom. I assume the runtime?

Reply:
I guess it's both the user and the runtime. The user should know that the runtime considers it only as a hint.

Maybe it's simpler to just say "The scheduling option is a hint." :)